ghzdac_cal/GHz_DAC_calibrate.py

```diff
@@ -26,9 +33,9 @@
 # GHz_DAC_calibrate is only a front-end, the actual calibration
 # routines are in ghzdac.calibrate
 
-import ghzdac
-import ghzdac.calibrate as calibrate
-import ghzdac.keys as keys
+import servers.ghzdac as ghzdac
+import servers.ghzdac.calibrate as calibrate
+import servers.ghzdac.keys as keys
```

Comment:
Are these imports correct? This assumes that the servers repo is cloned as `servers` and the parent directory is on the python path. If we want to actually put the code in this repo in a package, then I think we should reorganize this repo to match the structure in the `pylabrad` and `pyle` repos where there is a package *directory* in the repo with an `__init__.py` file, rather than putting the `__init__.py` at the repo top level, as here.